apps/service-portal/src/hooks/useAuth/useAuth.ts

```diff
@@ -16,6 +16,7 @@ const useAuth = () => {
         type: ActionType.SetUserFulfilled,
         payload: user,
       })
+      localStorage.setItem('token', user.access_token)
```

Comment:
I think we have to avoid keeping tokens in the browser. It can be vulnarable to (XSS) attacks.